frontend/src/scenes/surveys/SurveyRepeatSchedule.tsx

```diff
@@ -18,7 +18,8 @@ function doesSurveyHaveDisplayConditions(survey: Pick<Survey, 'conditions'>): bo
         survey.conditions?.url ||
         survey.conditions?.selector ||
         (survey?.conditions?.deviceTypes?.length ?? 0) > 0 ||
-        (survey?.conditions?.seenSurveyWaitPeriodInDays ?? 0) > 0
+        (survey?.conditions?.seenSurveyWaitPeriodInDays ?? 0) > 0 ||
+        survey.conditions?.linkedFlagVariant
```

Comment:
I'm not sure if this should be here, its used by `doesSurveyHaveDisplayConditions`

Reply:
this is a helper function to show users if they put repeating surveys but don't add any conditions, as it'll be repeated

i don't think adding a flag variant will help with that, so let's remove